types/halogenium/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for halogenium 2.3
+// Project: https://github.com/kirillDanshin/halogenium#readme
+// Definitions by: Weslley Nascimento Rocha <https://github.com/WeslleyNasRocha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
Is this really the minimum version of TypeScript that is compatible with these types?

Reply:
ow no, i just put the version i was running, didn't knew this was the minimum version